dts/arm/qemu-virt/qemu-virt-a53.dtsi

```diff
@@ -29,6 +29,12 @@
 		};
 	};
 
+	uartclk: uart-clock {
```

Comment:
https://github.com/qemu/qemu/blob/d4f7d56759f7c75270c13d5f3f5f736a9558929c/hw/arm/virt.c#L242-L248

Maybe renaming it to `apb-pclk` would be more true to the QEMU virt machine definition?

Reply:
@stephanosio will rename to node name to match.  Going to leave the label as is for now so its maybe a bit more readable.

Reply:
Updated